README.md

```diff
@@ -80,6 +80,13 @@ If unspecified, no artifact will be created.
 
 Default: `''` (no artifact)
 
+### `customize_file_colors`
+
+User can customize color of file. need to be a Json String. it will replace the key-value pair of [src/language-colors.json].
```

Comment:
what about:

> You can customize the colors for specific file extensions. Key/value pairs will extend the [default colors](src/language-colors.json).

Reply:
Ok thanks very much, My English is not good, I will change it as you said